src/SprykerSdk/Zed/SprykGui/Business/Spryk/Spryk.php

```diff
@@ -96,7 +96,7 @@ public function runSpryk(string $sprykName, array $formData): string
         $commandLine = $this->getCommandLine($sprykName, $normalizedFormData);
         $process = $this->getProcess($commandLine);
         $process->setEnv([
-            'DEVELOPMENT_CONSOLE_COMMANDS' => true,
+            'DEVELOPMENT_CONSOLE_COMMANDS' => 'true',
```

Comment:
Or should this be `'1'` string etc?
Do they al work the same?

Reply:
It looks like we use it only in such way, but I can change to '1'
```
$config[ConsoleConstants::ENABLE_DEVELOPMENT_CONSOLE_COMMANDS] = (bool)getenv('DEVELOPMENT_CONSOLE_COMMANDS');

Reply:
Either way works
Just that `'false'` would not work this way, right? So people might be tempted to change it as string and would be surprised why it doesnt work then.